include/kodlab_mjbots_sdk/log.h

```diff
@@ -4,199 +4,575 @@
  * @brief Macros for logging to console
  * @date 2022-07-13
  *
- * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
- * 
- * @todo(ethanmusser) Implement stack trace.
+ * @copyright (c) Copyright 2022 The Trustees of the University of Pennsylvania.
+ * All rights reserved.
  *
- * This header provides a set of debug logging macros with adjustable
+ * The `log.h` header provides a set of debug logging macros with adjustable
  * logging severity levels.  In order of increasing severity, the levels are
- *  - 'DEBUG' (0)
- *  - 'INFO' (1)
- *  - 'WARN' (2)
- *  - 'ERROR' (3)
- *  - 'FATAL' (4)
- *  - 'NONE' (5)
- *  .
- * The minimum level for console output is set by defining `LOG_MIN_SEVERITY`
- * (default is `DEBUG`).
+ * `TRACE`, `DEBUG`, `INFO`, `NOTICE`, `WARN`, `ERROR`, and `FATAL`.
```

Comment:
Notice is a better more standard name.

Reply:
Haha yeah I did some googling to see what was conventional.